### ISSUE TITLE
chat participant info is not keyboard accessible

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#212839]
1. Use a mouse to hover over the chat participant
2. See info and an action to go to the marketplace
3. Try to access this info via the keyboard
4. 🐛 one cannot
